src/coreclr/src/vm/dllimport.cpp

```diff
@@ -6597,6 +6597,15 @@ VOID NDirect::NDirectLink(NDirectMethodDesc *pMD)
     // Loading unmanaged dlls can trigger dllmains which certainly count as code execution!
     pMD->EnsureActive();
 
+    {
+        LPVOID pvTarget = (LPVOID)PInvokeOverride::TryGetMethodImpl(pMD->GetLibNameRaw(), pMD->GetEntrypointName());
```

Comment:
"Try" semantics should return a bool, not a "truthy" type. I would drop the "Try" of update the signature to return a `BOOL` or `bool`.

Also let's add a stresslog entry here that indicates the method was redirected.

Reply:
It is fairly common for TryXX, MaybeGetXXX  to not involve `bool` when the result type itself can express an absence of value. Sometimes it results in more efficient code (esp. in C#). 
I see uses of either convention in existing code, but returning bool seems definitely more common.

`GetMethodImpl` sounds good then. 

Stress log is a good idea too.